apps/jupyter/run-jupyter.sh

```diff
@@ -42,19 +51,20 @@ else
         2>> "${SSLLOG}" || die "Failed to create ${HOST} certificate"
         echo "Created ${HOST} certificate"
     fi
-    SSLARG="--certfile=\"${SSLPFX}-cert.pem\" --keyfile=\"${SSLPFX}-key.pem\""
+    SSLARG=("--certfile=\"${SSLPFX}-cert.pem\"" "--keyfile=\"${SSLPFX}-key.pem\"")
 fi
 
 (
-    while [[ $(ls /app/.local/share/jupyter/runtime/jpserver-*-open.html 2>/dev/null | wc -w) = 0 ]]; do
+    while [[ $(find "${RUNTIME_DIR}" -type f -name 'jpserver-*-open.html' 2>/dev/null | wc -l) = 0 ]]; do
         echo "Waiting for jpserver-*-open.html to appear"
         sleep 1
     done
-    FILE="$(ls /app/.local/share/jupyter/runtime/jpserver-*-open.html)"
-    if [[ $(echo "${FILE}" | wc -w) != 1 ]]; then
-        echo "ERROR: got '${FILE}' for jpserver files"
-        ls /app/.local/share/jupyter/runtime
-        echo "ERROR: Multiple jpsterver-*-open.html files"
+    FILE="$(find "${RUNTIME_DIR}" -type f -name 'jpserver-*-open.html')"
```

Comment:
why change these from ls to find?  I think both are correct in the given usage.

Reply:
The big issue is if the * expands to something containing spaces, but also for uniformity.

Reply:
I don't see how that's an issue. Agree on uniformity, if we're going to use find in one place we should use it everywhere.